addons/mrp_account/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="res_config_settings_mrp_account_view_form" model="ir.ui.view">
```

Comment:
This doesn't follow the standard documentation naming scheme or what most people have already done for their inherited views. Doesn't matter since mgm's request to move this to the product.category but you should keep that in mind for future tasks.